modules/dag-l1/src/test/scala/org/tessellation/dag/l1/infrastructure/DatabaseSuite.scala

```diff
@@ -0,0 +1,29 @@
+package org.tessellation.dag.l1.infrastructure.db
+
+import cats.effect.IO
+
+import org.tessellation.dag.l1.config.types.DBConfig
+import org.tessellation.dag.l1.infrastructure.address.storage.AddressStorage
+import org.tessellation.schema.address.Address
+import org.tessellation.schema.balance.Balance
+
+import ciris.Secret
+import eu.timepit.refined.auto._
+import eu.timepit.refined.types.all.NonNegLong
+import weaver.SimpleIOSuite
+
+object DatabaseSuite extends SimpleIOSuite {
+  test("Test writing to the database.") {
+    val dbConfig = new DBConfig("org.sqlite.JDBC", "jdbc:sqlite:test.db", "sa", Secret(""))
```

Comment:
instead of writing to `test.db`, you can create a temporary directory, see: https://github.com/pathikrit/better-files#temporary-files

you're sure the test is working in a "separate environment" each time